diff --git a/src/TodoPage.qml b/src/TodoPage.qml
index cc5d0a3..407e383 100644
--- a/src/TodoPage.qml
+++ b/src/TodoPage.qml
@@ -273,7 +273,7 @@ Kirigami.ScrollablePage {
         Kirigami.Action {
             text: i18nc("@action:inmenu", "Help…")
             icon.name: "help-contents-symbolic"
-            onTriggered: Qt.openUrlExternally("help:/komodo")
+            onTriggered: Qt.openUrlExternally("https://docs.kde.org/trunk_kf6/en/komodo/komodo/index.html")
             enabled: pageStack.layers.depth <= 1
             shortcut: StandardKey.HelpContents
         },
