diff --git a/src/TodoPage.qml b/src/TodoPage.qml
index cc5d0a3..c8e8913 100644
--- a/src/TodoPage.qml
+++ b/src/TodoPage.qml
@@ -56,7 +56,6 @@ Kirigami.ScrollablePage {
             cardsListView.enabled = true;
         }
         modality: Qt.ApplicationModal
-        nameFilters: ["Text files (*.txt)"]
     }
 
     Dialogs.FileDialog {
@@ -74,7 +73,7 @@ Kirigami.ScrollablePage {
         fileMode: Dialogs.FileDialog.SaveFile
         modality: Qt.ApplicationModal
         defaultSuffix: "txt"
-        nameFilters: ["Text files (*.txt)"]
+        selectedFile: "todo.txt"
     }
 
     Kirigami.Dialog {
